src/IW/Effects/Download.hs

```diff
@@ -0,0 +1,27 @@
+{-# LANGUAGE AllowAmbiguousTypes  #-}
+
+module IW.Effects.Download where
+
+import Network.HTTP.Client (Manager, Request (..), Response (..), httpLbs)
+import Network.HTTP.Types (Status (..))
+
+import IW.App (App (..), grab, Has, WithError)   
+import IW.Core.Url (Url (..)) 
+
+
+class Monad m => MonadDownload m where
```

Comment:
Let's add some documentation here or at the top of the module :slightly_smiling_face:

Reply:
@vrom911 For sure :+1: